docs/source/ttnn/ttnn/usage.rst

```diff
@@ -86,7 +86,8 @@ Basic Examples
 
 .. code-block:: bash
 
-    # To print currently executing ttnn operations
+    # To print currently executing ttnn operations and synchronize/block after each call
```

Comment:
I would also change the section header. Maybe to something like
8.Making Calls Blocking and Enabling Logging
or
9. Synchronizing After Each Call and Enabling Logging